oamapps/postConfigure/postConfigure.cpp

```diff
@@ -353,74 +320,11 @@ int main(int argc, char* argv[])
         exit(1);
     }
 
-    //check mysql port changes
-    string MySQLPort;
-
-    try
-    {
-        MySQLPort = sysConfig->getConfig(InstallSection, "MySQLPort");
-    }
-    catch (...)
-    {}
-
-    if ( mysqlPort == oam::UnassignedName )
-    {
-        if ( MySQLPort.empty() || MySQLPort == "" )
-        {
-            mysqlPort = "3306";
-
-            try
-            {
-                sysConfig->setConfig(InstallSection, "MySQLPort", "3306");
-            }
-            catch (...)
-            {}
-        }
-        else
-            mysqlPort = MySQLPort;
-    }
-    else
-    {
-        // mysql port was providing on command line
-        try
-        {
-            sysConfig->setConfig(InstallSection, "MySQLPort", mysqlPort);
-        }
-        catch (...)
-        {}
-    }
-
 	// perform single server install
 	cout << endl << "Performing the Single Server Install." << endl << endl;
 
 	//setup to Columnstore.xml file for single server
-	singleServerConfigSetup(sysConfig);
-
-	//module ProcessConfig.xml to setup all apps on the pm
-	if ( !updateProcessConfig() )
-		cout << "Update ProcessConfig.xml error" << endl;
-
-	try
-	{
-		sysConfig->setConfig(InstallSection, "SingleServerInstall", "y");
-		sysConfig->setConfig(InstallSection, "ServerTypeInstall", "2");
-	}
-	catch (...)
-	{
-		cout << "ERROR: Problem setting SingleServerInstall from the MariaDB ColumnStore System Configuration file" << endl;
-		exit(1);
-	}
-
-	if ( !writeConfig(sysConfig) )
-	{
-		cout << "ERROR: Failed trying to update MariaDB ColumnStore System Configuration file" << endl;
-		exit(1);
-	}
-
-	setSystemName();
-	cout << endl;
-
-	system(cmd.c_str());
+    //singleServerConfigSetup(sysConfig);
```

Comment:
leaving this commented out instead of removing for now. singleServerConfigSetup configures our standard single server